MapboxDirections.xcodeproj/project.pbxproj

```diff
@@ -632,6 +635,7 @@
 				39D04EFCE1A7300374AA5FBF /* Frameworks */,
 				CA9D333DD4D637ED30DA1B5D /* [CP] Embed Pods Frameworks */,
 				B146922AAE71DC82A56BBA60 /* [CP] Copy Pods Resources */,
+				43F82C8111AB3A1D45F1197F /* Embed Pods Frameworks */,
```

Comment:
You’d have to either `pod deintegrate` or delete all the CocoaPods-generated phases first, then `pod install` again. Or, if either of these “Embed Pods Frameworks” phases is empty, you can safely remove it.